docs/user/filesystem.md

```diff
@@ -218,6 +218,265 @@ dfuse           9.4G  326K  9.4G   1% /tmp/dfuse
+Failed to create unit file /tmp/dev-sda2.swap, as it already exists. Duplicate entry in /etc/fstab?
+# # Please ignore this Failed Error
+
+$ cat /tmp/tmp-root_dfuse.mount
+# Automatically generated by systemd-fstab-generator
+
+[Unit]
+SourcePath=/etc/fstab
+Documentation=man:fstab(5) man:systemd-fstab-generator(8)
+Before=local-fs.target
+After=daos_agent.service
+Requires=daos_agent.service
+
+[Mount]
+Where=/tmp/root_dfuse
+What=/usr/bin/dfuse
+Type=fuse3
+Options=pool=admin_pool,container=admin_cont,auto,x-systemd.requires=daos_agent.service
+$ cp -rf /tmp/tmp-root_dfuse.mount /usr/lib/systemd/system/
+$ systemctl daemon-reload
+$ systemctl status tmp-root_dfuse.mount
+● tmp-root_dfuse.mount - /tmp/root_dfuse
+   Loaded: loaded (/etc/fstab; generated)
+   Active: inactive (dead)
+    Where: /tmp/root_dfuse
+     What: /usr/bin/dfuse
+     Docs: man:fstab(5)
+           man:systemd-fstab-generator(8)
+
+Sep 16 21:43:14 wolf-170.wolf.hpdd.intel.com systemd[1]: tmp-root_dfuse.mount: Succeeded.
+$ systemctl start tmp-root_dfuse.mount
+$ df -h | grep fuse
+dfuse                         537G  5.1G  532G   1% /tmp/root_dfuse
+$ ls -l /tmp/root_dfuse/
+total 0
+$ systemctl status tmp-root_dfuse.mount
+● tmp-root_dfuse.mount - /tmp/root_dfuse
+   Loaded: loaded (/etc/fstab; generated)
+   Active: active (mounted) since Fri 2022-09-16 22:58:40 UTC; 2min 10s ago
+    Where: /tmp/root_dfuse
+     What: dfuse
+     Docs: man:fstab(5)
+           man:systemd-fstab-generator(8)
+    Tasks: 63 (limit: 1648282)
+   Memory: 55.7M
+   CGroup: /system.slice/tmp-root_dfuse.mount
+           └─10228 /usr/bin/dfuse /tmp/root_dfuse -o rw pool=admin_pool container=admin_cont dev suid
+
+Sep 16 22:58:39 wolf-170.wolf.hpdd.intel.com systemd[1]: Mounting /tmp/root_dfuse...
+Sep 16 22:58:40 wolf-170.wolf.hpdd.intel.com systemd[1]: Mounted /tmp/root_dfuse.
+$ systemctl stop tmp-root_dfuse.mount
+$ df -h | grep fuse
+$ systemctl status tmp-root_dfuse.mount
+● tmp-root_dfuse.mount - /tmp/root_dfuse
+   Loaded: loaded (/etc/fstab; generated)
+   Active: inactive (dead) since Fri 2022-09-16 23:01:22 UTC; 7s ago
+    Where: /tmp/root_dfuse
+     What: /usr/bin/dfuse
+     Docs: man:fstab(5)
+           man:systemd-fstab-generator(8)
+    Tasks: 0 (limit: 1648282)
+   Memory: 776.0K
+   CGroup: /system.slice/tmp-root_dfuse.mount
+
+Sep 16 22:58:39 wolf-170.wolf.hpdd.intel.com systemd[1]: Mounting /tmp/root_dfuse...
+Sep 16 22:58:40 wolf-170.wolf.hpdd.intel.com systemd[1]: Mounted /tmp/root_dfuse.
+Sep 16 23:01:22 wolf-170.wolf.hpdd.intel.com systemd[1]: Unmounting /tmp/root_dfuse...
+Sep 16 23:01:22 wolf-170.wolf.hpdd.intel.com systemd[1]: tmp-root_dfuse.mount: Succeeded.
+Sep 16 23:01:22 wolf-170.wolf.hpdd.intel.com systemd[1]: Unmounted /tmp/root_dfuse.
+```
+
+#### 4> systemd to mount the dfs container during system boot.
+
+  Same systemd file mention in previous example is used to mount the fuse during system power ON.
```

Comment:
Nitpic, power ON and system boot are not the sam thing, you mean system boot throughout.

Reply:
I will update to specific "system power cycle"